src/EventSubscriber/ApmCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ApmCommand implements EventSubscriberInterface
+{
+    private $enabled;
+    public function __construct()
+    {
+        $this->enabled = class_exists(\BlackfireProbe::class, false);
```

Comment:
```suggestion
        $this->enabled = method_exists(\BlackfireProbe::class, 'startTransaction');
```

This would avoid issues with older versions of the probe.